spec.html

```diff
@@ -26222,10 +26240,12 @@ <h1>Runtime Semantics: CreateDynamicFunction ( _constructor_, _newTarget_, _kind
             1. Let _sourceString_ be the string-concatenation of _prefix_, *" anonymous("*, _P_, 0x000A (LINE FEED), *") {"*, _bodyString_, and *"}"*.
             1. Let _sourceText_ be ! UTF16DecodeString(_sourceString_).
             1. Perform the following substeps in an implementation-dependent order, possibly interleaving parsing and error detection:
-              1. Let _parameters_ be the result of parsing ! UTF16DecodeString(_P_), using _parameterGoal_ as the goal symbol. Throw a *SyntaxError* exception if the parse fails.
-              1. Let _body_ be the result of parsing ! UTF16DecodeString(_bodyString_), using _goal_ as the goal symbol. Throw a *SyntaxError* exception if the parse fails.
+              1. Let _parameters_ be ParseText(! UTF16DecodeString(_P_), _parameterGoal_).
+              1. If _parameters_ is a List of errors, throw a *SyntaxError* exception.
+              1. Let _body_ be ParseText(! UTF16DecodeString(_bodyString_), _goal_).
+              1. If _body_ is a List of errors, throw a *SyntaxError* exception.
               1. Let _strict_ be ContainsUseStrict of _body_.
-              1. If any static semantics errors are detected for _parameters_ or _body_, throw a *SyntaxError* exception. If _strict_ is *true*, the Early Error rules for <emu-grammar>UniqueFormalParameters : FormalParameters</emu-grammar> are applied.
+              1. If _strict_ is *true*, apply the Early Error rules for <emu-grammar>UniqueFormalParameters : FormalParameters</emu-grammar>.
```

Comment:
```suggestion
              1. If _strict_ is *true*, apply the Early Error rules for <emu-grammar>UniqueFormalParameters : FormalParameters</emu-grammar> to _parameters_.
```

Reply:
Yeah, it's interesting, the status quo doesn't specify what the rules should be applied to. There's similar wording elsewhere in the spec, but it's always in the context of an Early Errors clause, where the 'target' is presumably the instance of the header production that's currently being examined.

So, yup, I'll add this.